.gitignore

```diff
@@ -11,3 +11,5 @@ node_modules
 .idea
 *.iml
 .vscode
+
+docs/index.html
```

Comment:
Built assets are ignored, but you can still test them locally with `npx serve ./docs`